documentation/guides/migrating-from-v3-to-v4.md

```diff
@@ -73,10 +73,11 @@ Polaris now supports multiple languages and ships with [many translations](https
 
 // new
 import translations from '@shopify/polaris/locales/en.json';
-
 <AppProvider i18n={translations}>
 ```
 
+If you wish to dynamically load translations based on a provided locale see the [AppProvider README](https://github.com/Shopify/polaris-react/blob/master/src/components/AppProvider/README.md#using-translations) for more information.
```

Comment:
I think it'd be good to mention `@shopify/react-i18n` up front in this case. Here's an option (the link needs to be added, sorry I couldn't find it): 

```suggestion
If you use [`@shopify/react-i18n`](link) and want to dynamically load translations based on a provided locale,  see the [AppProvider README](https://github.com/Shopify/polaris-react/blob/master/src/components/AppProvider/README.md#using-translations) for more information.
```

Reply:
Andrew mentioned this too - I'm 👍 on this change